css-brochure-form/styles.css

```diff
@@ -0,0 +1,40 @@
+.form-container {
+  width: 500px;
+  margin: 0 auto;
+}
+
+.learn {
+  font-size: x-large;
+  color: orangered;
+  text-align: center;
+}
+
+.input-box,
+.dropdown {
+  border-radius: 6px;
+  border-color: lightgrey;
+  width: 500px;
+  margin-bottom: 15px;
+  padding: 10px;
+}
+
+label {
+  display: block;
+  padding-left: 10px;
+  font-size: smaller;
+}
+
+.submit {
+  border-radius: 15px;
+  border-color: orangered;
+  border-style: ;
```

Comment:
If you're not going to assign anything to `border-style`, then no need to include it